docs/intro/contributing.txt

```diff
@@ -369,15 +369,15 @@ Writing the code for your ticket
 
 Next we'll be adding the ``make_toast()`` function.
 
-Navigate to the ``django/`` folder and open the ``shortcuts.py`` file. At the
+Navigate to the :file:`django/` folder and open the ``shortcuts.py`` file. At the
```

Comment:
Why can't (all?) the cases in this file be `:source:`? (They're actual files in django/django repository.) 🤔